diesel/src/pg/types/floats/mod.rs

```diff
@@ -43,12 +43,12 @@ impl Error for InvalidNumericSign {
 impl FromSql<types::Numeric, Pg> for PgNumeric {
     fn from_sql(bytes: Option<&[u8]>) -> Result<Self, Box<Error+Send+Sync>> {
         let mut bytes = not_none!(bytes);
-        let ndigits = try!(bytes.read_u16::<BigEndian>());
-        let mut digits = Vec::with_capacity(ndigits as usize);
+        let num_digits = try!(bytes.read_u16::<BigEndian>());
```

Comment:
Clippy found this because the names `ndigits` and `digits` were to similar, I was actually glad that it did. I had to think about it for a bit to understand what was meant here and I initially thought this was `n` represented as bytes (but there was no `n`). I think `num_*` is still a bit weird (but was just 3 chars to add), I would probably write `digit_count`.